tests/activityDetection.test.js

```diff
@@ -1,10 +1,30 @@
-// const activityDetection = require('../js/activityDetection/activityDetection.js');
-// const electron = require('./__mocks__/electron');
+const electron = require('./__mocks__/electron');
+const childProcess = require('child_process');
 
-xdescribe('Tests for Activity Detection', function() {
+let activityDetection;
 
-    beforeAll(function () {
-        activityDetection.setActivityWindow(120000, electron.ipcRenderer);
+describe('Tests for Activity Detection', function() {
+
+    var originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000;
+
+    beforeAll(function (done) {
+        childProcess.exec('npm rebuild --runtime=electron --target=1.2.2 --disturl=https://atom.io/download/atom-shell --build-from-source', function (err) {
```

Comment:
that target works?  even though our electron version is 1.6.11???  have you run all unit tests locally on mac and windows?